src/style.css

```diff
@@ -25,5 +41,26 @@ body {
 }
 
 .card-size {
-  width: 18rem;
+  width: 22rem;
+}
+
+.btn {
+  background-color: bisque;
+  color: rgb(49, 32, 11);
+}
+
+.btn:hover {
+  background-color: rgb(235, 200, 158);
+  color: rgb(32, 20, 6);
+}
+
+.card-img-top {
+  min-height: 200px;
+  max-height: 200px;
```

Comment:
This is a question for adding to my information. instead of these two, Why haven't you used `height: 200px`?